tests/js/client/server_parameters/test-export-shard-usage-metrics-bytes-per-shard-cluster.js

```diff
@@ -0,0 +1,1744 @@
+/*jshint globalstrict:false, strict:false */
+/* global getOptions, assertEqual, assertNotEqual, assertTrue, assertFalse */
+
+////////////////////////////////////////////////////////////////////////////////
+/// DISCLAIMER
+///
+/// Copyright 2024, ArangoDB Inc, Cologne, Germany
+///
+/// Licensed under the Apache License, Version 2.0 (the "License");
+/// you may not use this file except in compliance with the License.
+/// You may obtain a copy of the License at
+///
+///     http://www.apache.org/licenses/LICENSE-2.0
+///
+/// Unless required by applicable law or agreed to in writing, software
+/// distributed under the License is distributed on an "AS IS" BASIS,
+/// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+/// See the License for the specific language governing permissions and
+/// limitations under the License.
+///
+/// Copyright holder is ArangoDB Inc, Cologne, Germany
+///
+/// @author Jan Steemann
+/// @author Copyright 2024, ArangoDB Inc, Cologne, Germany
+////////////////////////////////////////////////////////////////////////////////
+
+if (getOptions === true) {
+  return {
+    'server.export-shard-usage-metrics': "enabled-per-shard",
+  };
+}
+
+const jsunity = require('jsunity');
+const db = require('@arangodb').db;
+const { getDBServers, getCoordinators } = require("@arangodb/test-helper");
+const request = require("@arangodb/request");
+
+function testSuite() {
+  const baseName = "UnitTestsCollection";
+  let nextCollectionId = 0;
+
+  let getUniqueCollectionName = () => {
+    return baseName + nextCollectionId++;
+  };
+
+  let getRawMetrics = function() {
+    let lines = [];
+    getDBServers().forEach((server) => {
+      let res = request({ method: "GET", url: server.url + "/_admin/usage-metrics" });
+      lines = lines.concat(res.body.split(/\n/).filter((l) => l.match(/^arangodb_collection_requests_bytes_(read|written)_total/)));
+    });
+    return lines;
+  };
+
+  let getParsedMetrics = function(database, collection) {
```

Comment:
All those helper functions are also being used in: `test-export-shard-usage-metrics-per-shard-cluster.js`. 
(without -bytes- in it). We could think about moving them into a dedicated metrics helper js module.